package.json

```diff
@@ -25,7 +25,9 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "pretest": "yarn run lint",
-    "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache"
+    "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
+    "coverage": "react-scripts test --env=jsdom --coverage",
```

Comment:
It seems like you have already setup the jest config to default to collecting coverage. If that is so, I don't see the point in adding a script for this.

Reply:
The configuration for the jest config, wasnt showing the coverage when I tested locally. maybe its a problem with create-react-app. I added this script as I would get coverage because of that. Maybe I remove the configuration with jest config.

Reply:
If that was the behavior with the current state of the jest config, it seems to me that the coverage is being generated but not shown. If that is the case, personally I might prefer to see this particular script named `test:show_coverage`, or perhaps the coverage-defaulting removed from the jest config and made explicit everywhere it is wanted.

If in fact the current jest config generates but does not display coverage reports in the test run, that would potentially point to some mistakes made by the jest team. Given their API, `collectCoverage` should enable both generation and display.

OK, so digging in a little further, you shouldn't have removed the 'text' `coverageReporter` in the config, and really you [don't actually need to specify reporters at all](https://jestjs.io/docs/en/configuration#coveragereporters-array-string), as they'll default to `["json", "lcov", "text", "clover"]`. 

I'm not aware of any liability to having clover there needlessly, though for build parsimony's sake I wouldn't object to seeing it fall off. Nonetheless the cost to us is so small and you don't know what tools your devs will be wanting to use. 

Personally I think I would axe the clover reporter and put it back in the event a developer on the team asked for it.

I wasn't sure what was motivating the reporter-axing decision but I didn't ask about it on our first pass through. The text reporter needs to be restored.